silhouette/Graphtec.py

```diff
@@ -138,11 +138,13 @@
 # Negative Acnoledge - Returns device tool setup
 CMD_NAK = '\x15'
 
+SILHOUETTE_CAMEO4_TOOL_EMPTY = 0
 SILHOUETTE_CAMEO4_TOOL_BLADE = 1
 SILHOUETTE_CAMEO4_TOOL_AUTOBLADE = 2
 SILHOUETTE_CAMEO4_TOOL_DEEPCUT = 3
 SILHOUETTE_CAMEO4_TOOL_2MM_CRAFTBLADE = 4
```

Comment:
Furthermore I hava a 3mm Kraftblade, and it is also reported back by the Cameo4 as `4`, so I would just write
```
SILHOUETTE_CAMEO4_TOOL_KRAFTBLADE = 4
```

Reply:
Ok, so its only kraftblade

Reply:
exactly